src/tailwind/plugins/theme/gc-colors.js

```diff
@@ -1,11 +1,15 @@
 /**
- * SAA colors
+ * Giving Campaign colors
  */
 module.exports = function () {
   return {
-    'gc-electric-blue': {
-      DEFAULT: '#505EEC',
-      light: '#6B77F5',
-    },
+    flamingo: '#ECC7CD',
+    fuchsia: '#E31C79',
+    lavender: '#C5B4E3',
+    lime: '#DBE442',
+    periwinkle: '#485CC7',
+    'robins-egg': '#77C5D5',
+    sapphire: '#005776',
+    slate: '#4E4B48',
```

Comment:
The brand colors from the agency's brand guide
![Screenshot 2023-04-06 at 3 21 11 PM](https://user-images.githubusercontent.com/42749717/230506948-cba26163-31cb-4c13-ac72-306c451af9fa.png)